tutorials/llm/llama-3/README.rst

```diff
@@ -17,6 +17,6 @@ This repository contains jupyter notebook tutorials using NeMo Framework for Lla
    * - `Llama 3.1 Law-Domain LoRA Fine-Tuning and Deployment with NeMo Framework and NVIDIA NIM <./sdg-law-title-generation>`_
      - `Law StackExchange <https://huggingface.co/datasets/ymoslem/Law-StackExchange>`_
      - Perform LoRA PEFT on Llama 3.1 8B Instruct using a synthetically augmented version of Law StackExchange with NeMo Framework, followed by deployment with NVIDIA NIM. As a pre-requisite, follow the tutorial for  `data curation using NeMo Curator <https://github.com/NVIDIA/NeMo-Curator/tree/main/tutorials/peft-curation-with-sdg>`__.
-   * - `Llama 3.1 WikiText Pruning and Distillation with NeMo Framework <./pruning-distillation>`_
```

Comment:
Line 5/5 fix capitalization

This repository contains Jupyter Notebook tutorials using the NeMo Framework for LLama-3 and LLama-3.1 models by Meta.